skills_asses/src/app/home/home.component.css

```diff
@@ -0,0 +1,38 @@
+header{
+    background: linear-gradient(90deg, #35142C 0%, #612550 25%, #497284 75%);
+     /* box-shadow: 2px 2px #00000040; */
+    display: flex;
```

Comment:
Note: you can use bootstrap classes like: `d-flex`  `align-items-center` `justify-content-between`